src/test/ui/nll/return_from_loop.stderr

```diff
@@ -22,7 +22,7 @@ LL |         my_struct.field.push_str("Hello, world!");
 LL | }
    | - first borrow ends here
 
-error[E0499]: cannot borrow `my_struct.field` as mutable more than once at a time (Mir)
+error[E0499]: cannot borrow `my_struct.field` as mutable more than once at a time (Body)
```

Comment:
Pretty much none of these tests need to be changed (they're caused by the `BorrowckMode` change, which isn't necessary)